<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SM_GUI_Golowasch_et_al_1999</class>
 <widget class="QDialog" name="SM_GUI_Golowasch_et_al_1999">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>294</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Golowash et al. 1999 synapse parameters</string>
  </property>
  <widget class="QLabel" name="label_101">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>70</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>s_fast</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="double_gSlow">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_29">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Fast</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_77">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>113</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>g (in uS):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_85">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>40</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="double_vthFast">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>37</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>25</number>
   </property>
  </widget>
  <widget class="QSpinBox" name="double_vthSlow">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>97</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>15</number>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="double_k1">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>160</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
   <property name="value">
    <double>14.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_76">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>53</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>g (in uS):</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_30">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>82</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Slow</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_106">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_103">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>61</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>s_slow</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="double_gFast">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>50</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="singleStep">
    <double>0.100000000000000</double>
   </property>
   <property name="value">
    <double>0.200000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_96">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>163</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>k2</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="double_k2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="decimals">
    <number>4</number>
   </property>
   <property name="singleStep">
    <double>0.001000000000000</double>
   </property>
   <property name="value">
    <double>4.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_95">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>163</y>
     <width>21</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>k1</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="double_sFast">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>64</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>5</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_87">
   <property name="geometry">
    <rect>
     <x>348</x>
     <y>100</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>84</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>19</y>
     <width>291</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_78">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V_slow</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_67">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>40</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>V_fast</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="double_sSlow">
   <property name="geometry">
    <rect>
     <x>289</x>
     <y>127</y>
     <width>51</width>
     <height>24</height>
    </rect>
   </property>
   <property name="maximum">
    <number>100</number>
   </property>
   <property name="value">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_105">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>70</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>%</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_accept">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>250</y>
     <width>82</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Accept</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
